### ISSUE TITLE
Define particle direction by an angle?

### ISSUE THREAD

**Issue by selamhabibi** [tsparticles/tsparticles#1594]
**Which version are you using?**
1.27.0

**Is the latest version affected?**

*You can checkout latest version [here](https://www.npmjs.com/package/tsparticles)*

- [x] Yes
- [ ] No

**Which library are you using?**

- [ ] Vanilla JS (`tsparticles`)
- [ ] React (`react-tsparticles`)
- [ ] Vue.js 2.x (`particles.vue`)
- [ ] Vue.js 3.x (`particles.vue3`)
- [ ] Angular (`ng-particles`)
- [ ] Svelte (`svelte-particles`)
- [ ] Preact (`preact-particles`)
- [ ] Inferno (`inferno-particles`)
- [x] jQuery (`jquery-particles`)

If I want to set my own direction for particles I have the following options:
 
  ```javascript
"none"
"top"
"top-right"
"right"
"bottom-right"
"bottom"
"bottom-left"
"left"
"top-left"
  ```

Is there a solution that I can specify an angle for the direction to determine any direction between 0 and 359 degrees?

**Comment by matteobruni** [comment]
I thought it was possible but I found out that it's not. I'm going to fix this in the upcoming `1.28.0` version.

It seems the `angle` property inside the `move` section is bugged.

**Comment by matteobruni** [comment]
I just released the `1.28.0` version, you can now specify an angle in degrees in `particles.move.direction` or the old values ("top", "right", "none", etc.)

**Comment by selamhabibi** [comment]
Thank you very much! Everything works as described.
